libs/ui/src/components/fields/input-text-field/InputTextField.tsx

```diff
@@ -31,6 +33,7 @@ export function InputTextField({
       label={label}
       variant={variant}
       helperText={error ? error : ' '}
+      required={required}
       {...register(id)}
```

Comment:
don't use this for fields, instead use the react-hook-form's `FormProvider` and `useController` to avoid passing a `register` prop to all future field ui components